easybuild/easyconfigs/c/CAFE5/CAFE5-5.0.0-GCC-10.2.0.eb

```
@@ -0,0 +1,40 @@
+# EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Max Voit
+
+easyblock = 'ConfigureMake'
+
+name = 'CAFE5'
+version = '5.0.0'
+
+homepage = 'https://github.com/hahnlab/CAFE5'
+
+description = """Software for Computational Analysis of gene Family Evolution
+
+The purpose of CAFE is to analyze changes in gene family size in a way that
+accounts for phylogenetic history and provides a statistical foundation for
+evolutionary inferences. The program uses a birth and death process to model
+gene gain and loss across a user-specified phylogenetic tree. The distribution
+of family sizes generated under this model can provide a basis for assessing the
+significance of the observed family size differences among taxa.
+"""
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+#toolchainopts = {'openmp': True}
+
+source_urls = ['https://github.com/hahnlab/CAFE5/releases/download/v5.0']
+sources = ['%(name)s-%(version)s.tar.gz']
+checksums = ['9e6dfd27fb915ba927f007e1f851630ff0547589b8196bd5f4606f3b23a78118']
+
+builddependencies = []
+
+dependencies = []
+
+install_cmd = 'mkdir %(installdir)s/bin && cp bin/cafe5 %(installdir)s/bin/cafe5 && cp -r examples %(installdir)s'
+
+sanity_check_paths = {
+    'files': ['bin/cafe5',
+              ],
+    'dirs': ['bin', 'examples'],
+}
+
```

We also like to have sanity check commands that verify if compiled executables can actually be executed, so usually something like `<executable> --version` or `<executable> --help` is already sufficient. The first seems to work for this application, so please add the following:
```suggestion

sanity_check_commands = ['cafe5 --help']

```